samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -121,16 +121,16 @@ The behavior of the commands vary depending on the hardware configuration and Kc
 
 * DTM with front-end module support:
 
-   * The official ``0x09`` DTM command when the :kconfig:option:`CONFIG_DTM_POWER_CONTROL_AUTOMATIC` Kconfig option is enabled, which is a default state.
+   * The official ``0x09`` DTM command when the :ref:`CONFIG_DTM_POWER_CONTROL_AUTOMATIC <CONFIG_DTM_POWER_CONTROL_AUTOMATIC>` Kconfig option is enabled, which is a default state.
```

Comment:
This sentence seems to be lacking what the command does. Or, in other words, there should be something between "DTM command" and "when". Could also probably be written the same as the Line 133 as they seem to be contrasting options.

Also, "the default" instead of "a default".